docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,341 @@
++++
```

Comment:
Please change the front matter to YAML. We're doing this as we come across them in the docs.